pages/desktop/consumer_pages/base.py

```diff
@@ -84,6 +86,29 @@ class HeaderRegion(Page):
         def is_user_logged_in(self):
             return self.is_element_visible(*self._settings_toggle_locator)
 
+        @property
+        def categories_name(self):
+            return self.selenium.find_element(*self._categories_toggle_locator).text
+
+        def open_categories_menu(self):
+            categories = self.selenium.find_element(By.ID, 'header--categories')
+            self.selenium.find_element(*self._categories_toggle_locator).click()
+            WebDriverWait(self.selenium, self.timeout).until(self.element_settled(categories))
+
+        class element_settled(object):
```

Comment:
I like this method - it's something we have needed for some time. I wonder, though, whether it would be more appropriate to put it in `page.py`?